### PR TITLE
APPS-128: Added refresh token handling 

### PR DESCRIPTION
**Pull request by Fleeck**
Also, updated README with step by step guide for ngrok

### CODE REVIEW

File: javascript/nodejs/server/routes/index.js [hunk 1]
```diff
@@ -7,7 +7,33 @@ var xml2js = require('xml2js');
...
+  request({ // request access_token
+      url: config.tradeshiftAPIServerURL + 'auth/token',
+      method: 'POST',
+      headers: {
+        Authorization: 'Basic ' + base64.encode(config.clientId + ':' + config.clientSecret)
+      },
+      formData: { // sending our refresh data
+        grant_type: 'refresh_token',
+        refresh_token: req.session.refresh_token,
+        scope: 'Kind.nodesample.0.0.1' // VendorId.AppId.Version
```
Comment (jbarmash) [inline javascript/nodejs/server/routes/index.js]: This should not be hard coded.

**Comment by jbarmash** [comment]
I see that you are getting the refresh token, but not actually using it to refresh the access code after some time.  That's the whole point of the refresh code.  We should show an example of that.

**Comment by tstapleton** [comment]
you should just use [passport](https://www.npmjs.com/package/passport). i don't know why anyone would actually want to implement this on their own. and i say that with experience - https://github.com/Tradeshift/workato-integration-app/pull/4.

**Comment by jbarmash** [comment]
@Fleeck  - I agree with Tom, it means we don't need to write custom code if we don't need to.

**Pull request closed without merging**